scargo/file_generators/templates/scargo.toml.j2

```diff
@@ -107,6 +107,7 @@ flash-start = 0x08000000
 [scargo]
 console-log-level = "INFO"
 file-log-level = "WARNING"
-update-exclude = [
+update-exclude = []
 
-]
+[docker-compose]
```

Comment:
please add 
[docker-compose]
ports = [] 

section above [scargo] section. Presently during .lock file generation, version is added at the end of the file and will be added to wrong section (probably need to be fixed in the future)

Reply:
No, I fixed it back in January. It was a bug even before, because for example the user can change the order of tables.

You have a habit of stating wrong things in your reviews as if they were facts. It would be much nicer to read and respond to your comments if you used questions instead, e.g. "Do we still have the bug where the version is added to the end of the lock file regardless of the order of sections?".